examples/99-advanced/planets.py

```diff
@@ -0,0 +1,119 @@
+"""
+.. _planets_example:
+
+3D Earth and Celestial Bodies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Creates high-resolution renderings of the Earth and the major celestial bodies
+in our solar system for astrodynamics applications.
+This example is inspired by `planet3D-MATLAB <https://github.com/tamaskis/planet3D-MATLAB>`_ .
+
+
+Note that this example author doesn't know anything about astronomy. There may
+be inaccuracies in the representation. The purpose of this example is for
+inspiration only.
+
+"""
+import pyvista
+from pyvista import examples
+
+###############################################################################
+# Celestial bodies with the Milky Way in the background.
+
+
+# Light of the sun in the +x axis direction.
+
+light = pyvista.Light()
+light.set_direction_angle(30, -20)
+
+# Load planet
+
+# https://tamaskis.github.io/files/Visualizing_Celestial_Bodies_in_3D.pdf
+# Mercury's radius is 2439.0 km
+mercury = examples.planets.load_mercury(radius=2439.0, lat_resolution=150, lon_resolution=300)
+# Venus's radius is 6052.0 km
+venus = examples.planets.load_venus(radius=6052.0, lat_resolution=150, lon_resolution=300)
+# Mars's radius is 3397.2 km
+mars = examples.planets.load_mars(radius=3397.2, lat_resolution=150, lon_resolution=300)
+# Jupiter's radius is 71492.0 km
+jupiter = examples.planets.load_jupiter(radius=71492.0, lat_resolution=150, lon_resolution=300)
+# Saturn's radius is 60268.0 km
+saturn = examples.planets.load_saturn(radius=60268.0, lat_resolution=150, lon_resolution=300)
+# Saturn's rings range from 7000.0 km to 80000.0 km from the surface of the planet
+inner = 60268.0 + 7000.0
+outer = 60268.0 + 80000.0
+saturn_ring_alpha = examples.planets.load_saturn_ring_alpha(inner=inner, outer=outer, c_res=50)
+# Uranus's radius is 25559.0 km
+uranus = examples.planets.load_uranus(radius=25559.0, lat_resolution=150, lon_resolution=300)
+# Neptune's radius is 24764.0 km
+neptune = examples.planets.load_neptune(radius=24764.0, lat_resolution=150, lon_resolution=300)
```

Comment:
I'd consider defining `kwargs = dict(lat_resolution=150, lon_resolution=300)` to reduce some duplication, using calls as
```py
neptune = examples.planets.load_neptune(radius=24764.0, **kwargs)
```

Reply:
OK. It works in my local.
```suggestion
# Set resolution.
kwargs = dict(lat_resolution=150, lon_resolution=300)
mercury = examples.planets.load_mercury(radius=2439.0, **kwargs)
# Venus's radius is 6052.0 km
venus = examples.planets.load_venus(radius=6052.0, **kwargs)
# Mars's radius is 3397.2 km
mars = examples.planets.load_mars(radius=3397.2, **kwargs)
# Jupiter's radius is 71492.0 km
jupiter = examples.planets.load_jupiter(radius=71492.0, **kwargs)
# Saturn's radius is 60268.0 km
saturn = examples.planets.load_saturn(radius=60268.0, **kwargs)
# Saturn's rings range from 7000.0 km to 80000.0 km from the surface of the planet
inner = 60268.0 + 7000.0
outer = 60268.0 + 80000.0
saturn_ring_alpha = examples.planets.load_saturn_ring_alpha(inner=inner, outer=outer, c_res=50)
# Uranus's radius is 25559.0 km
uranus = examples.planets.load_uranus(radius=25559.0, **kwargs)
# Neptune's radius is 24764.0 km
neptune = examples.planets.load_neptune(radius=24764.0, **kwargs)
# Pluto's radius is 1151.0 km
pluto = examples.planets.load_pluto(radius=1151.0, **kwargs)
```